tests/k8s/1-036_validate_role_rolebinding_for_source_namespace/02-sourceNamespace_with_wildcard_pattern.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-1
```

Comment:
Check if the labels are correctly applied.

Reply:
@anandf ,Thanks, I have updated the e2e test to check for labels in ns,  which is mentioned in argocd.spec.sourceNamespace.